docs/cheaha/migration_overview.md

```diff
@@ -0,0 +1,194 @@
+1. Tunable data sink
+    1. With Ceph, we are able to tune which files are offloaded from GPFS based on user behavior and activity without interfering with ongoing analysis or burdening the researcher with quickly moving data to another storage system.
+    1. This allows us a powerful option to address potential GPFS storage constraints that has not been available previously.
+
+### Tiered Storage Summary
+
+In summary, **users should not take tiered storage into account when using Cheaha**.
+
+- There will be no changes in user experience when traversing a directory tree as file stubs will appear as normal files from the user's perspective.
+- Slight delays when fetching old data should be expected as the data is migrated from Ceph to GPFS. See [below](#initial-interaction-with-files)
+- Tiered storage is NOT back-up storage
+- Tiered storage IS a new tool to improve storage needs on Cheaha.
+- See notes on [quotas](#quotas) below
+
+## GPFS 5 Notes Post-Migration
+
+### Initial Interaction With Files
+
+After migration to GPFS 5, loading a given file will be slow during the initial read but will return to normal for all subsequent reads. This is an expected behavior of our new tiered file system compared to our GPFS 4 system. This will be especially evident during startup of interactive apps from the web portal and during activation and use of virtual environments such as `conda` due to the large number of files these tools use. Please be patient when starting an app such as Jupyter Notebook or reading an especially large file for the first time post-migration due to the delay caused by initial file reads.
+
+### Jobs Requeued Prior to Migration
+
+As mentioned in the [migration procedure](#migration-procedure), jobs running at the time accounts are put on hold will be requeued and held. This is not a pause, it is essentially a resubmission of that job under the same job ID. For most software, jobs restarting like this will not pose issues. The job will start from the beginning of the script and either perform the same tasks as previously until reaching the end or will continue from where it left off if the tool being used has that as an option and the script has accounted for it (rare across all software, see machine learning checkpointing as an example).
+
+Some software have checks built-in to make sure previously generated data are not being overwritten accidentally. If these software detect some amount of standard outputs already exist, they may fail as opposed to overwriting the existing data. As example of this is the MRI analysis software FreeSurfer. This is a hazard when requeueing running jobs which will immediately cancel any command being run in the job and exit. Once you have been notified of your migration time, please be cognizant of potential effects of requeuing your planned jobs if you're not certain they will finish before the migration begins.
+
+### GPFS 5 Compute Nodes
```

Comment:
Would it make sense to communicate using the "two cutover" model we've developed?

It may help to periodically reiterate, in each section/subsection, that this information is intended to be useful during the migration, and finalized information will be communicated for after the migration.